dashboard/public/i18n/en.json

```diff
@@ -25,7 +25,16 @@
     "username": "Username",
     "undefined": "undefined",
     "password": "Password",
+    "confirm-password": "Confirm Password",
     "new-password": "New Password",
+    "password-strength": "",
```

Comment:
- Keep the keys values in the JSON file in sync. In en.json line 29 is new-password whereas in hi,json line 37 is new-password

Reply:
pardon me for the inconsistency, just a typo.. will fix it asap.